### PR TITLE
Default SV callers to Manta (WES). Fix #3206

### LINKED ISSUE #3206: WES SVs default to "all callers" - MIP tag interaction issue?
All randomly drawn WES SV vars look like they were called by all callers. I would assume they are rather not tagged properly, and this is a default?

Screenshot 2022-03-03 at 16 48 49

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug. See Clinical-Genomics/MIP#1977.

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. load a wes MIP 10 case
2. note SV call filter status: should now be manta:PASS rather than all callers PASS.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: scout/parse/variant/callers.py [hunk 1]
```diff
@@ -48,4 +48,10 @@ def parse_callers(variant, category="snv"):
             filter_status = "Filtered - {}".format(filter_status.replace(";", " - "))
         callers["gatk"] = filter_status
 
+    if category == "sv" and not raw_info or other_info:
+        filter_status = "Pass"
+        if variant.FILTER is not None:
+            filter_status = "Filtered - {}".format(filter_status.replace(";", " - "))
+        callers["manta"] = filter_status
```
Comment (northwestwitch) [inline scout/parse/variant/callers.py:55]: That's because if the variant is called then it means that has passed in manta, right?

Comment (northwestwitch) [reply scout/parse/variant/callers.py:55]: It still feels a bit arbitrary to me, especially thinking about other facilities that might have different pipelines..

Comment (dnil) [reply scout/parse/variant/callers.py:55]: It is very arbitrary. I am not fond of it, but it follows from the same pattern we have for the SNVs with at least MIP<11. Lund use the more sensible caller tags so they should be fine. And for others, I don’t think seeing manta:pass is any worse than all callers:pass…

File: scout/parse/variant/callers.py [hunk 2]
```diff
@@ -48,4 +48,10 @@ def parse_callers(variant, category="snv"):
             filter_status = "Filtered - {}".format(filter_status.replace(";", " - "))
         callers["gatk"] = filter_status
 
+    if category == "sv" and not raw_info or other_info:
+        filter_status = "Pass"
+        if variant.FILTER is not None:
+            filter_status = "Filtered - {}".format(filter_status.replace(";", " - "))
```
Comment (northwestwitch) [inline scout/parse/variant/callers.py:54]: 👍🏻

**Review by northwestwitch — APPROVED** [review]
👍🏻 

Perhaps let's load a test case before merging?

File: scout/parse/variant/callers.py (hunk 2 above)
Comment (dnil) [reply scout/parse/variant/callers.py:54]: Agreed!

**Comment by dnil** [comment]
Meh, that's silly. I should have checked the VCFs first. Even the one-caller wes sv vcfs have 'set=Intersection' so in scout terms they will have `raw_info` `call` as `Intersection`. I'll close this.

**Pull request closed without merging**